gslib/tests/test_hmac.py

```diff
@@ -18,17 +18,28 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import re
 import boto
+import datetime
+import os
+import re
 
+from gslib import cloud_api_delegator
+from gslib.commands import hmac
 from gslib.project_id import PopulateProjectId
 import gslib.tests.testcase as testcase
 from gslib.tests.testcase.integration_testcase import SkipForS3
 from gslib.tests.testcase.integration_testcase import SkipForXML
 from gslib.tests.util import SetBotoConfigForTest
+from gslib.tests.util import SetEnvironmentForTest
 from gslib.tests.util import unittest
+from gslib.third_party.storage_apitools import storage_v1_messages as apitools_messages
```

Comment:
Is this still necessary? Ditto for the other new imports

Reply:
no, let me delete them

Reply:
Done.